kuiper-game/Domain/GameTime.cs

```diff
@@ -1,35 +1,43 @@
 using System;
+using Microsoft.VisualBasic;
 
-public struct GameTime
+namespace Kuiper.Domain
 {
-    private static readonly DateTime gameStartDate = new DateTime(2078, 1, 1);
-    private const long tickAccelerationConstant = 7; //1 real day, is 7 game days
-    public static DateTime RealStartTime {get; set;}
+    public readonly struct GameTime
```

Comment:
Consider adding comparison overrides to check one gametime against another, easily. (could be done with implicit operators too). - So when you want to do something like
if (ledger.saved > gametime.now())

Reply:
yeah, good idea. Will do that 👍